introduction.md

```diff
@@ -1,3 +1,6 @@
 Introduction of Ionut Verzea to Dojo Rules repo.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?